test/test_slacko.ml

```diff
@@ -0,0 +1,174 @@
+open Lwt
+open OUnit2
+
+open Slounit
+open Abbrtypes
+
+
+let token_str =
```

Comment:
I'd just match on exception. I think we do that somewhere anyway (and depend on a recent enough OCaml version) and I think it looks nice and coherent.